### ISSUE TITLE
runAnlysis

### ISSUE THREAD

**Issue by lhusova** [lhusova/diplomovka#1]
I'm running on ROOT 6 and I can't run the macro runAnalysis.C . I get error, that all used tasks are unknown or undeclared, but when i copy the macro into the terminal it works. And could you navigate me, how should I rewrite the part with runlists, that it will look better.

**Comment by mvala** [comment]
I will need to be able to run it and reproduce it. Attach directory or provide it in git

**Comment by lhusova** [comment]
Do you thing the source root file?

**Comment by mvala** [comment]
All macros

**Comment by lhusova** [comment]
Everything is already there, the other tasks I read from AliRoot/AliPhysics. You'll find it in the directory data.

**Comment by mvala** [comment]
@lhusova i understand that every task is from aliroot or aliphysics. I need to reproduce error you are getting with your macros. So i need your macros to run ```root runAnalysis.C```

**Comment by lhusova** [comment]
all macros are uploaded, you can find then in the directory data. you need AddMyTask.C and AliAnalysisTaskMyTask.h and .cxx

**Comment by mvala** [comment]
Ok I tested it with aliroot/aliphysics from cvmfs. It works. What is not working for you. I am not getting idea ROOT6 in here? AliPhysics doesn't support ROOT6 yet. You would have to do all Add macros by hand to support ROOT6 loading. And what do you mean i am running ROOT6. Did you compile aliroot with ROOT6?

**Comment by lhusova** [comment]
Yes I compiled Aliroot with ROOT6, because on new MacOs, there is no more support for ROOT5 in Aliroot. With ROOT5, there was no problem with this runMacro.

**Comment by mvala** [comment]
In this case there are 2 options. 
1. I need mac to see and debug
2. you can use docker with centos7 inside. I can show you how to do it on monday meeting
